components/st-microelectronics/landpatterns.stanza

```diff
@@ -257,3 +257,55 @@ public pcb-landpattern UFQFPN48:
                       tol(0.4, 0.1),   ;Lead size in x direction:   L
                       tol(0.25, 0.05), ;Lead size in y direction:   b
                       GeneralChamferedRectangle(5.6, 5.6, [0.5 0. 0. 0.]))
+
+public val STM32-LAND-PATTERNS = [
```

Comment:
This table includes all the land patterns in this file.

Reply:
This is not something we should support - I'm not aware of any situation where multiple pins should be mapped to the same pad

Reply:
ok chatted with mike this is middleware and at the schematic symbol level 1 pad will not be mapped to more than 1 schematic symbol pin